test/erc20_gateway/erc20_cogateway/confirm_deposit.js

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use-strict';
```

Comment:
Move this above license.

Reply:
Done.

Reply:
@sarvesh-ost I am not sure why it has been asked to move above license. There is not agreement there, however, in most of the  files it's after license (or it's not there at all). Do you see a point to have it as a first statement in the file?